.github/workflows/publish-package.yml

```diff
@@ -0,0 +1,38 @@
+name: publish distributions
+on:
+  push:
+    branches:
+    - master
```

Comment:
on master as well? why not only on tags?

Reply:
if you tag on non-master, you don't want it to run.

Reply:
This is to trigger the TestPyPI releases. The PyPI releases are still protected with

```yaml
      if: startsWith(github.ref, 'refs/tags')
```

@kratsg I'm heavily basing this off of `stare` so let me know if I'm missing any other reason. The reason I gave is why I kept it like you had it.